jogo42-cb/main.c

```diff
@@ -73,7 +81,41 @@ int main(void)
         gs->jog.rot = Vector2Angle(gs->jog.pos, PosWorldDoCursor(gs));
 
         // Mover inimigo
-        // TODO
+        if(CheckCollisionCircles((Vector2){gs->inim.pos.x, gs->inim.pos.y}, 160, (Vector2){gs->jog.pos.x, gs->jog.pos.y}, RAIO_JOG))
```

Comment:
`gs->inim.pos` já é um `Vector2`, então `(Vector2){gs->inim.pos.x, gs->inim.pos.y}` tem valor igual e é redundante. O mesmo vale para `gs->jog.pos`.

```suggestion
        if(CheckCollisionCircles(gs->inim.pos, 160, gs->jog.pos, RAIO_JOG))
```